### PR TITLE
Extend per-game configuration options

### PR DESCRIPTION
**Pull request by AbandonedCart**

### CODE REVIEW

File: core/nullDC.cpp [hunk 1]
```diff
@@ -129,15 +129,26 @@ void* webui_th(void* p)
 cThread webui_thd(&webui_th,0);
 #endif
 
+#if defined(_ANDROID)
+int reios_init_value;
```
Comment (baka0815) [inline core/nullDC.cpp]: Why introduce this here?
Can't you do ``reios_init_value = dc_init();`` from Android?

**Comment by AbandonedCart** [comment]
While not every regular setting has been made available, this removes the block on adding all but reios itself, since that would require a duplicate init and should not be to inconvenient to toggle universally.

File: core/nullDC.cpp (hunk 1 above)
Comment (AbandonedCart) [reply core/nullDC.cpp]: No. That defeats the purpose.

Comment (AbandonedCart) [reply core/nullDC.cpp]: The first half of the init is required to obtain valid value for the disk info, but also contains the return values (in the regular init). The second half of the init would not run if a return value is set, but needs to be run after the disk info.
If you do not store the values of the first half, the second half will run unchecked. If you run the second half before the values are obtained, none of the settings are applied.

Comment (baka0815) [reply core/nullDC.cpp]: And why do this only on Android?
Wouldn't that be good to do on other platforms too?

Comment (AbandonedCart) [reply core/nullDC.cpp]: It is on other platforms.

**Comment by AbandonedCart** [comment]
screenshot_20180826-115319

🥇

**Comment by AbandonedCart** [comment]
... definitely shouldn't be taking Travis 13 minutes to mark a successful build successful...

**Pull request merged**